mtags/src/main/scala-2/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -16,10 +16,10 @@ abstract class PcCollector[T](
     cursor = None
   )
   val pos: Position = unit.position(params.offset)
```

Comment:
If we change the parameter `VirtualFileParams` we can match here if it's an OffsetParams. If it's not OffsetParams we search for all symbols otherwise we use soughtSymbol?